src/components/genes/GenesSubscriber.js

```diff
@@ -54,7 +53,6 @@ export default function GenesSubscriber(props) {
   const numGenes = geneList.length;
 
   function setGeneSelectionAndColorEncoding(newSelection) {
-    setCellSetSelection([]);
```

Comment:
I understand the motivation behind removing this (i.e having both cell set and gene selection at the same time for the histogram) but two things jump out at me:
1. Without a histogram, this might look odd and also be tricky with saving state i.e from: #778 #776 and #775
2. We still can't jump back to the gene view without selection a different gene first and then going back to the gene (i.e just clicking on the selected gene doesn't bring you back to the "gene" view)

Should we change the genes view from radio buttons then?  I don't really feel confident about saying what we should do here, but if you have a vision/plan here, I won't stand in the way

Reply:
I have fixed the issue with selecting already-selected genes/cell sets, allowing their re-selection to trigger a new selection event. I also added a new option to the dropdown for the scatterplot and spatial plot to explicitly set the `cellColorEncoding` coordination value.

![Screen Shot 2021-01-21 at 3 24 36 PM](https://user-images.githubusercontent.com/7525285/105408521-11e4e880-5bfd-11eb-8404-7b07151eb987.png)